client/data/agency-dashboard/use-fetch-dashboard-sites.ts

```diff
@@ -38,18 +38,16 @@ interface FetchDashboardSitesArgsInterface {
 	agencyId?: number;
 }
 
-const useFetchDashboardSites = ( args: FetchDashboardSitesArgsInterface ) => {
-	const {
-		isPartnerOAuthTokenLoaded,
-		searchQuery,
-		currentPage,
-		filter,
-		sort,
-		perPage = 20,
```

Comment:
We don't set a default value here, so it is easier to match queryKey later with an optimistic update.